src/server/DBMetadata/classes/processedField.js

```diff
@@ -1,9 +1,11 @@
+const util = require("../../util");
+
 class ProcessedField {
-  constructor(col, tblIdx, fieldIdx) {
+  constructor(col, tblIdx, fieldIdx, translateColumnType) {
```

Comment:
I like that you made this a dependency and didn't just include the file as it we're a helper library +1

Reply:
@laueian easier to switch out translation functions that way